push2dockerhub.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
can we move this to deployment

Reply:
See previous comments about what the deployment directory is for. If you like we can create a new top level directory for storing build related scripts - want to pick a name?

Reply:
See above

Reply:
Done.

Reply:
Looks like the file is still in root.